paf-mvp-core-js/src/crypto/signer.ts

```diff
@@ -13,13 +14,16 @@ export interface SignatureStringBuilder<U> {
  * U = Unsigned type (used for getting signature input)
  */
 export class Signer<U> {
+  protected logger = new Log('Signer', 'red');
+
   /**
    * @param ecdsaPrivateKey the private key that will be used to sign
    * @param definition defines how to get input string for signing
    */
   constructor(private ecdsaPrivateKey: PrivateKey, protected definition: SignatureStringBuilder<U>) {}
 
   sign(inputData: U): string {
+    this.logger.Debug('Sign', inputData);
```

Comment:
Do we consider the logged value as sensitive data?

Reply:
Interesting question.
Since from the beginning we consider that data is "sensitive", but not "personal", I guess logging locally is acceptable.